docs/sources/about.md

```diff
@@ -76,33 +81,51 @@ You should run the Static mode Kubernetes operator when:
 
-  * **Debuggability**: You need to more easily debug configuration issues using
-    a UI.
+  * **Improved debugging**: You need to more easily debug configuration issues using a UI.
+
+  * **Full OpenTelemetry support**: Support for collecting OpenTelemetry metrics, logs, and traces.
+
+  * **PrometheusRule support**: Support for the PrometheusRule resource from the Prometheus Operator project for configuring Grafana Mimir.
+
+  * **Ecosystem transformation**: You need to be able to convert Prometheus and Loki pipelines to and from OpenTelmetry Collector pipelines.
+
+  * **Grafana Pyroscope support**: Support for collecting profiles for Grafana Pyroscope.
+
+#### Core telemetry
+
+|              | Prometheus Agent mode | Grafana Agent Static mode | Grafana Agent Operator | OpenTelemetry Collector | Grafana Agent Flow mode  |
+|--------------|-----------------------|---------------------------|------------------------|-------------------------|--------------------------|
+| **Metrics**  | Prometheus            | Prometheus                | Prometheus             | OTel                    | [Prometheus][], [OTel][] |
+| **Logs**     | No                    | Loki                      | Loki                   | OTel                    | [Loki][], [OTel][]       |
+| **Traces**   | No                    | OTel                      | OTel                   | OTel                    | [OTel][]                 |
+| **Profiles** | No                    | No                        | No                     | No                      | Pyroscope                |
 
-  * **Full OpenTelemetry support**: Support for collecting OpenTelemetry
-    metrics, logs, and traces.
+#### **OSS features**
 
-  * **PrometheusRule support**: Support for the PrometheusRule resource from
-    the Prometheus Operator project for configuring Grafana Mimir.
+|                          | Prometheus Agent mode | Grafana Agent Static mode | Grafana Agent Operator | OpenTelemetry Collector | Grafana Agent Flow mode |
+|--------------------------|-----------------------|---------------------------|------------------------|-------------------------|-------------------------|
+| **Kubernetes native**    | No                    | No                        | Yes                    | Yes                     | Yes                     |
+| **Clustering**           | No                    | No                        | No                     | No                      | No                      |
+| **Prometheus rules**     | No                    | No                        | No                     | No                      | Yes                     |
+| **Native Vault support** | No                    | No                        | No                     | No                      | Yes                     |
 
-  * **Ecosystem transformation**: You need to be able to convert Prometheus and
-    Loki pipelines to and from OpenTelmetry Collector pipelines.
+#### Grafana Cloud solutions
```

Comment:
It feels strange to me for these to be subheaders of Flow mode, since it also describes the features of all other nodes. Should there be a dedicated `h2` section for these?

Reply:
I moved the tables up to the top of the document and added a simple lead-in